javascript-functions-practice/get-property-value.js

```diff
@@ -0,0 +1,18 @@
+/* exported getPropertyValue */
+function getPropertyValue(obj, key) {
+
+  if (key === 'birthPlace') {
+    var props = Object.entries(obj);
+    return props[2][1];
+  }
+
+  if (key === 'occupation') {
+    var props2 = Object.entries(obj);
+    return props2[1][1];
+  }
+
+  if (key === 'name') {
+    var props3 = Object.entries(obj);
+    return props3[0][1];
+  }
```

Comment:
This is built for the test, not as a general purpose function. Please revise this to work with any object and property passed to the function

Reply:
Thank you Cody! I just fixed it.